cpf_segmentation/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM ubuntu:18.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update
+RUN apt-get install -y cmake pkg-config libeigen3-dev libopencv-dev libpcl-dev
+
+WORKDIR /app/src/src
+
+ADD ./src .
+
+WORKDIR /app/src/gco-v3.0
```

Comment:
Why do you change workdir every time? Is it possible to add not only to the current workdir?

Reply:
It decreases writing `RUN cd` and `RUN mkdir`. That's why many `WORKDIR`'s here
https://www.educative.io/edpresso/what-is-the-workdir-command-in-docker#:~:text=If%20the%20WORKDIR%20command%20is%20not%20written%20in%20the%20Dockerfile%2C%20it%20will%20automatically%20be%20created%20by%20the%20Docker%20compiler.%20Hence%2C%20it%20can%20be%20said%20that%20the%20command%20performs%20mkdir%20and%20cd%20implicitly.